asm/inst_other.go

```diff
@@ -54,11 +55,16 @@ func (fgen *funcGen) newFCmpInst(ident ir.LocalIdent, old *ast.FCmpInst) (*ir.In
 // newPhiInst returns a new IR phi instruction (without body but with type)
 // based on the given AST phi instruction.
 func (fgen *funcGen) newPhiInst(ident ir.LocalIdent, old *ast.PhiInst) (*ir.InstPhi, error) {
+	astFmfs := old.FastMathFlags()
 	typ, err := fgen.gen.irType(old.Typ())
 	if err != nil {
 		return nil, errors.WithStack(err)
 	}
-	return &ir.InstPhi{LocalIdent: ident, Typ: typ}, nil
+	fmfs := make([]enum.FastMathFlag, 0, len(astFmfs))
```

Comment:
This can be simplified, by using the existing `irFastMathFlags` function.

For example, see [asm/inst_binary.go](https://github.com/llir/llvm/blob/0d19cd0df110ed9c8fbcec42e3a0239456b15e2c/asm/inst_binary.go#L188):

```go
	// (optional) Fast math flags.
	inst.FastMathFlags = irFastMathFlags(old.FastMathFlags())
```